core/datetimes/ad_datetime.py

```diff
@@ -114,6 +114,8 @@ def to_ad_datetime(self):
         return self
 
     def __eq__(self, other):
+        if other is None:
+            return super(AdDatetime, self).__eq__(other)
```

Comment:
Changes to add this condition to allow django-dirty-fields to compare None/Null if there is None type set in date field from Core module. Without this the django-dirty-fields isn't able to compare None/Null with DateTime from core module (default django DateTime is able to compare with None/Null)